pkg/client/client.go

```diff
@@ -486,6 +486,17 @@ func (o *OptimizelyClient) GetProjectConfig() (projectConfig config.ProjectConfi
 	return projectConfig, nil
 }
 
+// GetOptimizelyConfig returns OptimizelyConfig object
+func (o *OptimizelyClient) GetOptimizelyConfig() (optimizelyConfig *entities.OptimizelyConfig, err error) {
+
+	projectConfig, err := o.ConfigManager.GetConfig()
```

Comment:
We can simply call `o.GetProjectConfig()` right? Also, let's make that method private since now everyone should be using this new one.